src/components/policy/policy_external/src/update_status_manager.cc

```diff
@@ -164,9 +165,15 @@ bool UpdateStatusManager::IsUpdatePending() const {
 }
 
 void UpdateStatusManager::ScheduleUpdate() {
+  LOG4CXX_AUTO_TRACE(logger_);
   ProcessEvent(kScheduleUpdate);
 }
 
+void UpdateStatusManager::PendingUpdate() {
+  LOG4CXX_AUTO_TRACE(logger_);
+  ProcessEvent(kPendingUpdate);
+}
+
 void UpdateStatusManager::ScheduleManualUpdate() {
```

Comment:
@LitvinenkoIra good place for **`LOG4CXX_AUTO_TRACE`**

Reply:
@mkorniichuk Done in 7d5222c6ca6e483113b51eae3332d9e94ec5444a